test/dialect/mssql/test_engine.py

```diff
@@ -256,19 +257,35 @@ def test_pyodbc_odbc_connect_ignores_other_values(self):
                 "larry",
                 "{moe",
                 "localhost",
+                3306,
                 "mydb",
             ),
             (
-                "DRIVER={foob};Server=localhost;"
+                "DRIVER={foob};Server=localhost,3306;"
```

Comment:
Super-nit-pick: This is

test/dialect/mssql/test_engine.py

but 3306 is the default port for mysql, not mssql. Maybe a better choice would be a value that is clearly not a default port, like 43210?

Reply:
Agh, thank you - my memory failing me.  That's corrected, although I take your point - omitting it where it's the default probably makes more sense.  I'll update that soon.